### ISSUE TITLE
Conditional colors

### ISSUE THREAD

**Issue by prfalken** [tessera-metrics/tessera#295]
The 'singlegraph' item is inspired by Tasseo and is wonderful !

However Tasseo offers the conditional colors feature, which allows you to set thresholds for which the graph colors would change.

This feature would be awesome in Tessera : for instance the graphs would switch to red if the value go over a specific threshold.

**Comment by clohtd** [comment]
Quick feedback; I would be interested in using the threshold in both manners.  I have charts and singlestats side by side on some pages, using the same query.  On those pages, I'd like to use the same query but only have the singlestat change color when over threshold (ignoring the threshold in the time series).  So perhaps create a property of any renderer which would allow you to ignore threshold behavior in the query?

**Comment by aalpern** [comment]
Yeah, allowing a per-item override of the per-query threshold should be straightforward.

**Comment by filippog** [comment]
:+1: this would be awesome to have!

other random idea, the same concept can be applied to graphs too: e.g. (single) time series can have their background split in chunks and each chunk is colored based on how close the value is to the threshold for that chunk. Or bar charts have individual bars colored based on how close the value is to the threshold. Also note that I have no idea how easy/hard it is to implement this in practice :)
